pkg/pgmodel/cache/cache.go

```diff
@@ -13,12 +14,16 @@ import (
 
 const (
 	DefaultMetricCacheSize = 10000
+
+	nullSchemaName = "__null__"
```

Comment:
why don't we just use an empty string here?

Reply:
I guess we could, I'll update.